.azure/azure-pipelines.yml

```diff
@@ -3,33 +3,34 @@ jobs:
 
 - job: 'Test'
   pool:
-    vmImage: 'macOS-10.13'
+    vmImage: 'macOS-latest'
   strategy:
     matrix:
-    # TODO: enable these if/when keyper supports python < 3.7
-    #  Python35:
-    #    python.version: '3.5'
-    #  Python36:
-    #    python.version: '3.6'
-      Python37:
-        python.version: '3.7'
-    maxParallel: 4
+      python38:
```

Comment:
If we aren't testing Python 3.7 any more, we should drop support in the pyproject.toml and bump the major version.